### ISSUE TITLE
Bug: Preprint not found

### ISSUE THREAD

**Issue by murkatr** [PREreview/rapid-prereview#143]
https://outbreaksci.prereview.org/10.21203/rs.3.rs-32796/v1 says "Not Found"

Here's the pdf: https://assets.researchsquare.com/files/rs-32796/v1/4a557e94-2c89-41ab-89c1-9e2f1e75bcbf.pdf

Here it is on Research Square: https://www.researchsquare.com/article/rs-32796/v1

**Comment by murkatr** [comment]
![Not Found](https://user-images.githubusercontent.com/64230315/87380021-71738480-c546-11ea-8ea1-1078c1e8edff.png)

**Comment by georgiamoon** [comment]
@murkatr does it work if you remove the `v1` ? I believe that OSrPRE can't take the version numbers on the DOIs.

**Comment by murkatr** [comment]
@georgiamoon Unfortunately, it still does not work when the v1 is removed. 

@leonardosfl As an update, @georgiamoon will be emailing Research Square about this. We may need to take a different approach with Research Square preprints. (Also connecting the related #89 issue here.)

**Comment by georgiamoon** [comment]
@murkatr Sorry for confusion — that email conversation is a bit unrelated to this issue. 

My understanding from the @jheretic is that the issue is ResearchSquare doesn't enable us to reference the hosted PDF the way we are doing for other preprint servers. 

At the moment I think the fix to this issue is to provide clearer feedback that the PDF can't be loaded, rather than that the PDF is not found. This is more similar to what I was suggesting in #75 which is another situation in which the user experience is affected by the unavailability of a preprint (in that case a 503 error).

The solution in #146 looks good to me as a way to address this for now! 

Another idea would be to have some template for the metadata we do have about a preprint, e.g. title, authors, abstract, doi, that could display on the right hand side of the UI, which is similar to how this is handled for bioRxiv and medRxiv preprints on prereview.org, e.g.:
- https://www.prereview.org/preprints/doi/10.31234/osf.io/7yzhd

**Comment by georgiamoon** [comment]
@leonardosfl FYI — We decided to clarify this issue vs #75 so that #75 is about better error handling and this issue is about improving the display of ResearchSquare preprints!

**Comment by leonardosfl** [comment]
@georgiamoon Just to mention, this also affects ResearchGate preprints: https://outbreaksci.prereview.org/10.13140/RG.2.2.33033.06242/1

**Comment by georgiamoon** [comment]
Yes — My understanding is that we need a way to handle each preprint server which is slightly different. Is that your sense as well @leonardosfl ? or is there a common solution for a few of them?

**Comment by leonardosfl** [comment]
@georgiamoon The issue with ResearchGate is different. We are able to get the PDF URL, but ResearchGate doesn't seem to allow us to embed the PDF, it redirects to the preprint page if we try do it (which is the current behavior). I tried proxying it using the API, but ResearchGate detects that we are not using a browser and returns a `403 Forbidden`.

Because of that, I think we'll have to use the approach you mentioned before:
> Another idea would be to have some template for the metadata we do have about a preprint, e.g. title, authors, abstract, doi, that could display on the right hand side of the UI, which is similar to how this is handled for bioRxiv and medRxiv preprints on prereview.org, e.g.:
---
Moving to ResearchSquare, I noticed that it does allow embedding of PDFs, so we just need to implement the code to fetch the PDF URL.

**Comment by dasaderi** [comment]
@leonardosfl @jheretic @harumhelmy @rudietuesdays is there any update on this issue?

**Comment by brymut** [comment]
Hi, is this still available for anyone to try out?
